docs/src/main/asciidoc/_admin-guide/subsystem-configuration/EE_Concurrency_Utilities_Configuration.adoc

```diff
@@ -156,6 +157,18 @@ The optional `hung-task-threshold` defines a runtime threshold value, in
 milliseconds, for tasks to be considered hung by the executor.
 A value of `0` will never consider tasks to be hung.
 
+The optional `hung-task-termination-period` defines the period, in
+milliseconds, for attempting the termination of hung tasks, by cancelling
+their execution, and interrupting their executing threads. Please note
+that the termination of a cancelled hung task is not guaranteed.
+A value of `0`, which is the default, deactivates the periodic
+cancellation of hung tasks. Management clients, such as the WildFly CLI,
+may still be used to manually attempt the termination of hung tasks:
+[source,options="nowrap"]
+----
+/subsystem=ee/managed-executor-service=other:terminate-hung-tasks
+----
+
```

Comment:
@emmartins I guess "The mandatory `core-threads`" should be changed to "`The optional `core-threads`":

In facts in `standalone-full-ha.xml` it's not present:

<managed-executor-service 
  name="default" 
  jndi-name="java:jboss/ee/concurrency/executor/default" 
  context-service="default" 
  hung-task-threshold="60000" 
  keepalive-time="5000"/>

 .... it's not relevant in this MR though

Reply:
Right, it was mandatory initially but think later it was changed to optional, and when not specified it's computed automatically.

Reply:
I have created a JIRA to fix that at https://issues.redhat.com/browse/WFLY-14297